.github/actions/deploy-beta/action.yml

```diff
@@ -62,6 +74,10 @@ runs:
           HRM_STATIC_KEY=${{ inputs.hrm-static-key }}
           SURVEY_WORKFLOW_SID=${{ inputs.survey-workflow-sid }}
           POST_SURVEY_BOT_CHAT_URL=${{ inputs.post-survey-bot-chat-url }}
+          ASELO_APP_ACCESS_KEY=${{ inputs.aselo-app-access-key }}
+          ASELO_APP_SECRET_KEY=${{ inputs.aselo-app-secret-key }}
+          S3_BUCKET=${{ inputs.aws-region }}
+          AWS_REGION=${{ inputs.s3-bucket }}
```

Comment:
Out of these 4 values, there is only one actually changing from one configuration to another: `S3_BUCKET`. Maybe we don't need `ASELO_APP_ACCESS_KEY`, `ASELO_APP_SECRET_KEY` and `AWS_REGION` as inputs.